### ISSUE TITLE
FALSE NEGATIVE: `MC_OVERRIDABLE_METHOD_CALL_IN_CONSTRUCTOR` misses an overridable setter call

### ISSUE THREAD

**Issue by rikki322** [spotbugs/spotbugs#4309]
Hi, I found a false negative in SpotBugs 4.10.3 when a constructor calls an overridable setter declared in the same class.

**Affected tool**

`SpotBugs 4.10.3`

**Affected checker**

`SpotBugs MC_OVERRIDABLE_METHOD_CALL_IN_CONSTRUCTOR`

**Minimal reproducer**

```java
class SpotbugsSonarqubeConstructorSetter {
  int value;

  SpotbugsSonarqubeConstructorSetter(int value) {
    setValue(value);
  }

  void setValue(int value) {
    this.value = value;
  }
}
```

**Reproduction command**

```bash
spotbugs -version
javac -d classes spotbugs-sonarqube-mc-overridable-call-fn-setter.java
spotbugs -textui -effort:max -low -visitors FindOverridableMethodCall classes
```

**Current behavior**

SpotBugs produces no `MC_OVERRIDABLE_METHOD_CALL_IN_CONSTRUCTOR` diagnostic.

**Expected behavior**

SpotBugs should report line 5 because `setValue()` is overridable during construction.
